cardano-cli/src/Cardano/CLI/Types/Output.hs

```diff
@@ -363,14 +363,65 @@ renderScriptCosts
   -- ^ Post execution cost calculation mapping of script witness
   -- index to execution units.
   -> Either PlutusScriptCostError [ScriptCostOutput]
-renderScriptCosts (UTxO utxo) eUnitPrices scriptMapping executionCostMapping =
+renderScriptCosts (UTxO utxo) eUnitPrices scriptMapping =
```

Comment:
`renderScriptCosts` and `renderScriptCostsWithScriptHashesMap` should be one function